folium/raster_layers.py

```diff
@@ -73,33 +76,29 @@ def __init__(self, tiles='OpenStreetMap', min_zoom=0, max_zoom=18,
         self._name = 'TileLayer'
         self._env = ENV
 
-        options = {
-            'minZoom': min_zoom,
-            'maxZoom': max_zoom,
-            'noWrap': no_wrap,
-            'attribution': attr,
-            'subdomains': subdomains,
-            'detectRetina': detect_retina,
-        }
+        options = {'minZoom': min_zoom, 'maxZoom': max_zoom, 'noWrap': no_wrap,
+                   'attribution': attr, 'subdomains': subdomains,
+                   'detectRetina': detect_retina}
+        if max_native_zoom:
+            options['maxNativeZoom'] = max_native_zoom
```

Comment:
This is fine but I am 99% sure you can just pass the option and, if `None`, the `json.dumps` will convert to `null` and everything will work as expected on the JS side. Then we'll have 1 new line in the options instead of the if clause.

Reply:
Unfortunately the map won't load with `"maxNativeZoom": null`. I understand you don't want the extra if clause. I solved it by using `'maxNativeZoom': max_native_zoom or max_zoom`.

Reply:
> Unfortunately the map won't load with `"maxNativeZoom": null`.

Ah! I did not test that. Thanks!

> I understand you don't want the extra if clause.

Actually the if-clause was OK in that case.

> I solved it by using `'maxNativeZoom': max_native_zoom or max_zoom`.

That is fine too. Thanks!